src/pages/index.jsx

```diff
@@ -40,11 +40,33 @@ export default function Home() {
           <div className="container">
             <h1>Who is using Fastify?</h1>
             <p>Fastify is proudly powering a large ecosystem of organisations and products out there.</p>
-            <p>
-              Discover <Link to="/organisations">more organisations using Fastify</Link>. Do you want your organisation
-              to <Link to="/organisations#how-to-be-featured-here">be featured here</Link>?
-            </p>
-            <Organisations maxItems={12} />
+          </div>
+
+          <div className="container">
+            <div className="row">
+              <div className="col col--6">
+                <div className="alternate-sponsors">
+                  <h3>Sponsors</h3>
+                  <p>
+                    Would you like to sponsor Fastify financially? Support us on{' '}
```

Comment:
```suggestion
                    Would you like to sponsor Fastify? Support us on{' '}
```
I think `financially` is redundant as we're already asking if the user if they would like to sponsor Fastify, which implies financial aid.

Reply:
The issue I wanted to solve is that we have these sponsors:

https://github.com/fastify/fastify?tab=readme-ov-file#sponsors

but they sponsored fastify with devs' time and not money.

I think we should normalize that section TBH

cc @mcollina

Reply:
I think both should certainly exist but there should be a clear distinction between pure financial support to Fastify team and all other forms of support such as dev time etc.